### DISCUSSION TITLE
qsv example use cases infographic built with Typst

### DISCUSSION THREAD

**Discussion by rzmk** [dathere/qsv#2230] [Ideas]
Added an infographic built with Typst (including source code) for a few use cases when using qsv here in the wiki: https://github.com/jqnatividad/qsv/wiki/qsv-cheat-sheet

**Comment by jqnatividad** [comment]
@rzmk, I moved the "cheat sheet" to its own page in the Wiki! 😄 

https://github.com/jqnatividad/qsv/wiki/qsv-cheat-sheet
